lambda/src/firebase-authorizer/firebase-authorizer.js

```diff
@@ -116,29 +117,26 @@ const checkUserAuthorization = async event => {
   }
 }
 
+const _handler = async event => {
+  initNFA({
```

Comment:
I think we should init only once (as needed) at the top level—but that's just an optimization, so ignore if it's annoying to implement.

Reply:
Oh, sounds good. I had thought we needed to init on each invocation.